hex/lib/dependabot/hex/file_fetcher.rb

```diff
@@ -73,6 +76,14 @@ def evaled_files
           fetch_file_from_host(path).tap { |f| f.support_file = true }
         end
       end
+
+      def required_files
```

Comment:
There's a bit of duplication between this and `evaled_files` now, I wonder if we could either:

- extract that into a helper method
- or, update the regex to match both patterns, and update the method name to reflect that it deals with both cases

Reply:
I like the 2nd over the 1st, especially since there is already the concept of supporting files. so, if the method was named something like `support_files`, then it would fit quite well, with getting both of these cases. I'll check back tomorrow to see if theres any more discussion on it

Reply:
I think I prefer that as well 👍 thanks for your work on this by the way 🎉